dev-server/documents/pdf/jstor.config.json

```diff
@@ -8,11 +8,16 @@
       },
       "item": {
         "title": "Populations of Thrips tabaci, with Special Reference to Virus Transmission",
-        "containerTitle": "Journal of Animal Ecology"
+        "subtitle": "A subtitle"
```

Comment:
This JSON configuration is used for a local dev-server testing page with faked JSTOR content. If you're curious, you can see it at http://localhost:3000/pdf/jstor